app/src/main/java/org/p2p/wallet/claim/model/ClaimDetails.kt

```diff
@@ -0,0 +1,12 @@
+package org.p2p.wallet.claim.model
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class ClaimDetails(
+    val willGet: ClaimFee,
```

Comment:
`amount`
And it's not `ClaimFee` right?

Reply:
heh it is fee and amount
i think i need to rename it in common way